README.md

```diff
@@ -46,7 +46,7 @@ Project: Messenger with translating(Bachelor work)
  - В даному проекті використовується нереляційна база даних Firabase.
  - Firebase — це набір інструментів для «створювання, покращення та розвитку вашого додатка», а інструменти, які він надає, охоплюють значну частину сервісів, які розробникам зазвичай доводиться створювати самостійно, але насправді вони не хочуть створювати, оскільки вони краще зосередитися на самому додатку. Це включає такі речі, як аналітика, аутентифікація, бази даних, конфігурація, зберігання файлів, push-повідомлення, і список можна продовжувати. Служби розміщуються в хмарі та масштабуються без будь-яких зусиль з боку розробника.
  - Для підключення Firebase до нашого проекту треба в файл app.module.ts заімпортувати відповідні модулі(AngularFireModule - для загального підключення БД, AngularFirestoreModule - для  підключення зберігання даних в колекціях, AngularFireStorageModule - для зберігання фото і інших файлів у бд, AngularFireAuthModule - для роботи з логінуванням і реєстрацією).
- - Для даного проекту було створено 1 колекцію (users) - це набір користувачів нашогої системи. Кожен користувач є записаний в колекцію за своїм id. В кожного користувача є такі поля: email, image, userName, contacts, id. Розберемо кожен детально. 
```

Comment:
What is this? what should I review?

Reply:
As I wrote in the readme file, I used a non-relational database - Firebase. It has no links and tables as such, and data is stored in collections - arrays of objects. And those in turn can contain any type of data(string, number, object, array, date ets.). And values from the database are returned in JSON format.